api/test/unit/views/test_media_views.py

```diff
@@ -49,36 +47,6 @@ def test_retrieve_query_count(api_client, media_type_config):
     assert res.status_code == 200
 
 
-@pytest.mark.django_db
-@pytest.mark.parametrize(
-    "path, expected_params",
-    [
-        pytest.param("tag/cat/", {"tag": "cat"}, id="tag"),
-        pytest.param("source/flickr/", {"source": "flickr"}, id="source"),
-        pytest.param(
-            "source/flickr/creator/cat/",
-            {"source": "flickr", "creator": "cat"},
-            id="source_creator",
-        ),
-    ],
-)
-def test_collection_parameters(path, expected_params, api_client):
```

Comment:
Are these removed because they're tested by schemathesis?

Reply:
I thought that we don't need these tests because no longer have extra endpoints, and we _do_ have the serializer tests that check that when a search request has the collection-related query parameters, we create the correct query.

Reply:
It's a bit different, unit vs integration level tests. Integration level tests would be nice for these complex endpoints because it's easier to build the real edge cases into the tests. Schemathesis might do that too, but I don't think this style of test is duplicative of the unit/serializer level tests. Just because we have serializer tests to test serializer output, doesn't mean those tests also ensure the serializer output integrates with the actual search requests themselves in all instances. Integration tests do that testing of the glue.

Incidentally, integration tests could obviate the need for some of those unit tests, but only reliably so if we had coverage collection.

With the complexity of the endpoints parameters, I think it's not a bad idea to have tests of the sort removed here that send a variety of query parameter configurations and assert at minimum whether the result is a successful search or not.